codegenerator/cli/src/config_parsing/chain_helpers.rs

```diff
@@ -329,7 +329,7 @@ impl Network {
             | Network::Amoy
             | Network::Crab
             | Network::Darwinia
-            | Network::Cyber => 200,
+            | Network::Cyber => DEFAULT_CONFIRMED_BLOCK_THRESHOLD,
```

Comment:
At some point we should do some research on these chains and set the ones that don't reorg to 0. Made an issue here: https://linear.app/envio/issue/IDX-666/set-confirmed-block-thresholds-for-known-chains

Reply:
Should I set it to 0 for Fuel? I didn't really know what the value is responsible for

Reply:
It's the threshold that we guarentee history for in the event that we need to rollback on a reorg. For eg. a typical reorg is 1-2 blocks on ethereum. Polygon once experienced a 160 block reorg. But it's very uncommon but we want the smallest reasonable + safe threshold for storing history.